steps/appium-server/0.0.4/step.yml

```diff
@@ -0,0 +1,42 @@
+title: Run Appium Server
+summary: |
+  Install and Run Appium Server
+description: |
+  Install and Run Appium Server on Bitrise to be able to run our mobile automated tests
+website: https://github.com/moatazeldebsy/bitrise-step-appium-server
+source_code_url: https://github.com/moatazeldebsy/bitrise-step-appium-server
+support_url: https://github.com/moatazeldebsy/bitrise-step-appium-server/issues
+published_at: 2021-08-27T16:04:04.474263+02:00
+source:
+  git: https://github.com/moatazeldebsy/bitrise-step-appium-server.git
+  commit: 6a427500aee5a429f61d1bda9632a7e30ec455d0
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- android
+- react-native
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: nodejs
+  apt_get:
+  - name: nodejs
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- appium_version: v1.21.0
```

Comment:
Are you sure it's working with the `v` prefix?

Reply:
yes, you can check the release here [https://github.com/appium/appium/releases](https://github.com/appium/appium/releases)